### DISCUSSION TITLE
Unable to run cwe_checker

### DISCUSSION THREAD

**Discussion by examplebaker** [fkie-cad/cwe_checker#292] [General]
Hello, 
How can I verify that I've correctly installed the CWE checker? When I try to run on a C binary I created for testing I get the following: 
```
$ cwe_checker testfile
Error: Ghidra could not be executed:
Not a directory (os error 20)
```
I installed rust and ghidra prior to installing cwe_checker with the make command **make all GHIDRA_PATH=/path/to/ghidra_folder** where I replaced the ghidra path with the outcome of **which ghidra** 

What steps can I take to troubleshoot?

**Comment by Enkelmann** [comment]
This should be an issue, not a discussion. I created issue #293 for it. See there for the answer to your question. :)
